### ISSUE TITLE
How to work this with Angular? Any sample for background notification working in angular

### ISSUE THREAD

**Issue by nisha-ann** [NativeScript/sample-android-background-services#37]

**Comment by hkurma** [comment]
I'm looking for implementation of the same in Angular. Can somebody help me out?

**Comment by hkurma** [comment]
@nisha-ann Did you find any solution on this or a work around?

**Comment by nisha-ann** [comment]
No...I am doing another part of my project, but I will have to implement this in the 2nd phase....I will do this by that time...like in 2 weeks

**Comment by nisha-ann** [comment]
Let's try to implement this...if you have done it plz share it....

**Comment by csimpi** [comment]
I'd be very interested in a solution, still couldn't find out how to use this with Angular
